### PR TITLE
Simple atomistic

### PR DESCRIPTION
**Pull request by samwaseda**
And this probably marks my first PR to `pyiron_contrib`.

### CODE REVIEW

File: pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+from pyiron_atomistics.atomistics.job.interactive import GenericInteractive
+import numpy as np
+
+class SimpleAtomistic(GenericInteractive):  # Create a custom job class
```
Comment (jan-janssen) [inline pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py:8]: It would be great to add a `to_hdf()` and `from_hdf()` method.

Comment (liamhuber) [reply pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py:8]: When you do this you might find it's convenient to change most (all?) fields to live on an `InputList`.

File: pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        self.neigh = None
+        self.potential = None
+        self._neighbor_chemical_symbols = None
+        self.uptodate = False
+
+    def interactive_initialize_interface(self):
+        pass
+
+    def validate_ready_for_run(self):
+        super(SimpleAtomistic, self).validate_ready_for_run()
```
Comment (liamhuber) [inline pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py:20]: nit: moderns `super` calls don't need the arguments (unless you have dual inheritance and want non-default resolution order)

File: pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py (hunk 1 above)
Comment (pmrv) [inline pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py:8]:
```suggestion
class SimpleAtomistic(GenericInteractive):
```

**Pull request closed without merging**